doc-content/enterprise-only/springboot/bus-app-swagger_proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='bus-app-swagger_{context}']
+= Enable Swagger documentation
+
+You can enable Swagger-based documentation for all endpoints
+available in the service project of your {PRODUCT_BA} business application.
+
+.Prerequisites
+* You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[business applications] web site.
+
+. Unzip the `<business-application>.zip` file and navigate to the `<business-application>/<business-application>-service` folder.
+. Open the service project `pom.xml` file in a text editor.
+. Add the following dependencies to service project `pom.xml` file and save the file.
++   
```

Comment:
Separated the UI dependency as it's own step, but still put the applications.properties files step after for flow. Let me know if this is not OK, @MarianMacik